src/ParsePushAdapter.js

```diff
@@ -32,9 +33,11 @@ export default class ParsePushAdapter {
           this.senderMap[pushType] = new APNS(pushConfig[pushType]);
           break;
         case 'android':
-        case 'fcm':
           this.senderMap[pushType] = new GCM(pushConfig[pushType]);
           break;
+        case 'fcm':
+          this.senderMap[pushType] = new FCM(pushConfig[pushType]);
```

Comment:
going to take this out for now... getting this code checked in and verified before it gets used is probably safer than making it the default